### PR TITLE
#13: fixed tests

### PR DESCRIPTION
**Pull request by nikitacherevko**

### CODE REVIEW

File: services/src/messaging/src/main/java/org/openkilda/messaging/info/InfoMessage.java [hunk 1]
```diff
@@ -76,9 +76,9 @@ public InfoMessage(@JsonProperty(PAYLOAD) final InfoData data,
      * @param timestamp     timestamp value
      * @param correlationId message correlation id
      */
-    public InfoMessage(@JsonProperty(PAYLOAD) final InfoData data,
-                       @JsonProperty(TIMESTAMP) final long timestamp,
-                       @JsonProperty(CORRELATION_ID) final String correlationId) {
+    public InfoMessage(final InfoData data,
```
Comment (carmine) [inline services/src/messaging/src/main/java/org/openkilda/messaging/info/InfoMessage.java]: Why remove @JsonProperty?

File: services/src/messaging/src/main/java/org/openkilda/messaging/Message.java [hunk 2]
```diff
@@ -99,8 +99,7 @@ public Message(@JsonProperty(TIMESTAMP) final long timestamp,
      * @param timestamp     message timestamp
      * @param correlationId message correlation id
      */
-    public Message(@JsonProperty(TIMESTAMP) final long timestamp,
-                   @JsonProperty(CORRELATION_ID) final String correlationId) {
+    public Message(final long timestamp, final String correlationId) {
```
Comment (carmine) [inline services/src/messaging/src/main/java/org/openkilda/messaging/Message.java]: Why remove @JsonProperty?

Comment (nikitacherevko) [reply services/src/messaging/src/main/java/org/openkilda/messaging/Message.java]: We have already created constructor which is responsible for the creation objects from json. Also we mark only one constructor with @JsonCreator and it parameters with @JsonPropertyr.
https://stackoverflow.com/questions/15931082/how-to-deserialize-a-class-with-overloaded-constructors-using-jsoncreator
This constructor we use just to create object from javacode (not deserializing), so we don't need it marked with @JsonCreator

File: services/src/messaging/src/main/java/org/openkilda/messaging/info/InfoMessage.java (hunk 1 above)
Comment (nikitacherevko) [reply services/src/messaging/src/main/java/org/openkilda/messaging/info/InfoMessage.java]: the same is here (please take a look at the comment bellow)

**Pull request merged**